src/client.py

```diff
@@ -45,25 +46,28 @@ def _login(email: str, password: str):
                 raise IncorrectCredentials
         return session
 
-    def get_classes(self, target_day: datetime):
+    def get_classes(self, target_day: datetime, family_id: str = None):
         response = self.session.get(
             classes_endpoint(self.box_name),
             params={
                 "box": self.box_id,
                 "day": target_day.strftime("%Y%m%d"),
-                "familyId": "",
+                "familyId": family_id,
```

Comment:
did you try in production what happens if we send familyId: null? (sorry I don't have access to aimharder anymore)

Reply:
It works on every scenario: when your account only has one member (familyId is empty then), when your account has several members and familyId is empty you use the main member and when you send the familyId you work with that member.